airbyte-api/src/main/openapi/config.yaml

```diff
@@ -895,21 +925,15 @@ components:
           items:
             $ref: "#/components/schemas/SourceRead"
     # SOURCE SPECIFICATION
-    SourceSpecificationId:
-      type: string
-      format: uuid
     SourceSpecification:
       description: The specification for what values are required to configure the source.
       example: { user: { type: string } }
     SourceSpecificationRead:
       type: object
       required:
-        - sourceSpecificationId
         - sourceId
-        - specification
+        - connectionSpecification
```

Comment:
the naming is confusing here and it introduces a new term `connection`
what about something like: `expectedConfigurations` or `configurations`